Tools/machines/karolina-it4i/karolina_gpu_warpx.profile.example

```diff
@@ -0,0 +1,66 @@
+# please set your project account
+export proj=""  # change me!
+
+# remembers the location of this script
+export MY_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+if [ -z ${proj-} ]; then echo "WARNING: The 'proj' variable is not yet set in your $MY_PROFILE file! Please edit its line 2 to continue!"; return; fi
+
+# required dependencies
+module load GCCcore/11.3.0
+module load CUDA/11.7.0
+module load CMake/3.23.1-GCCcore-11.3.0
+module load OpenMPI/4.1.4-GCC-11.3.0-CUDA-11.7.0
+
+# optional: for QED support with detailed tables
+module load Boost/1.79.0-GCC-11.3.0
+
+# optional: for openPMD and PSATD+RZ support
+module load OpenBLAS/0.3.20-GCC-11.3.0
+export CMAKE_PREFIX_PATH=${HOME}/sw/karolina/gpu/hdf5-1.14.1.2:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=${HOME}/sw/karolina/gpu/c-blosc-1.21.1:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=${HOME}/sw/karolina/gpu/adios2-2.8.3:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=${HOME}/sw/karolina/gpu/blaspp-master:$CMAKE_PREFIX_PATH
+export CMAKE_PREFIX_PATH=${HOME}/sw/karolina/gpu/lapackpp-master:$CMAKE_PREFIX_PATH
+
+export LD_LIBRARY_PATH=${HOME}/sw/karolina/gpu/hdf5-1.14.1.2/lib64:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=${HOME}/sw/karolina/gpu/c-blosc-1.21.1/lib64:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=${HOME}/sw/karolina/gpu/adios2-2.8.3/lib64:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=${HOME}/sw/karolina/gpu/blaspp-master/lib64:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=${HOME}/sw/karolina/gpu/lapackpp-master/lib64:$LD_LIBRARY_PATH
+
+# optional: CCache (not found)
+#module load ccache
+
+# optional: for Python bindings or libEnsemble
+module load Python/3.10.4-GCCcore-11.3.0-bare
+
+if [ -d "${HOME}/sw/karolina/gpu/venvs/warpx-gpu" ]
+then
+  source ${HOME}/sw/karolina/gpu/venvs/warpx-gpu/bin/activate
+fi
+
+# an alias to request an interactive batch node for one hour (TODO)
```

Comment:
Trying to find the equivalent aliases for PBS, I came up with this:

```
# an alias to request an interactive batch node for one hour
#   for parallel execution, start on the batch node: mpirun -n <number_of_processes> <command>
alias getNode="qsub -q qgpu -A $proj -l select=1:ncpus=32:ngpus=4 -l walltime=1:00:00 -I"

# an alias to run a command on a batch node for up to 30min
#   usage: runNode <command>
alias runNode='echo -e "#!/bin/bash\n#PBS -q qgpu\n#PBS -A $proj\n#PBS -l select=1:ncpus=32:ngpus=4\n#PBS -l walltime=0:30:00\nmpirun -n 4 $1" | qsub'
```

There doesn't seem to be a direct equivalent to `srun` so one is forced to use a shell script instead